iothub/device/src/DeviceClient.cs

```diff
@@ -30,13 +30,16 @@ private DeviceClient(InternalClient internalClient)
                 throw new ArgumentException("A module ID was specified in the connection string - please use ModuleClient for modules.");
             }
 
-            if (Logging.IsEnabled) Logging.Associate(this, this, internalClient, nameof(DeviceClient));
+            if (Logging.IsEnabled)
+            {
+                Logging.Associate(this, this, internalClient, nameof(DeviceClient));
+            }
         }
 
         /// <summary>
-        /// Creates a disposable, Amqp DeviceClient from the specified parameters
+        /// Creates a disposable, AMQP DeviceClient from the specified parameters
```

Comment:
nit: How about - `Creates a disposable DeviceClient from the specified parameters, that uses AMQP transport protocol.`